include/z64save.h

```diff
@@ -717,8 +719,12 @@ typedef enum {
 #define WEEKEVENTREG_20_80 PACK_WEEKEVENTREG_FLAG(20, 0x80)
 #define WEEKEVENTREG_21_01 PACK_WEEKEVENTREG_FLAG(21, 0x01)
 #define WEEKEVENTREG_21_02 PACK_WEEKEVENTREG_FLAG(21, 0x02)
-#define WEEKEVENTREG_21_04 PACK_WEEKEVENTREG_FLAG(21, 0x04)
-#define WEEKEVENTREG_21_08 PACK_WEEKEVENTREG_FLAG(21, 0x08)
+
+// Player has spoken to Goron Graveyard's gravemaker while in Goron form.
+#define WEEKEVENTREG_TALKED_TO_GORON_GRAVEMAKER_AS_GORON PACK_WEEKEVENTREG_FLAG(21, 0x04)
+// Goron aside Goron Graveyard has been thawed.
+#define WEEKEVENTREG_THAWED_GRAVEYARD_GORON PACK_WEEKEVENTREG_FLAG(21, 0x08)
```

Comment:
```suggestion
// Goron outside Goron Graveyard has been thawed.
#define WEEKEVENTREG_THAWED_GRAVEYARD_GORON PACK_WEEKEVENTREG_FLAG(21, 0x08)
```
Is this what you mean?

Also, there are two distinct events. First, you thaw the goron, then you talk to the goron. Which of the two sets this flag? If you thaw the goron but walk away without talking, is this flag set?

Reply:
You're right, this is set inside the message script, not directly as a a result of thawing. Will add "TALKED" prefix.